link/src/main/java/com/stripe/android/link/account/DefaultLinkAccountManager.kt

```diff
@@ -155,6 +169,18 @@ internal class DefaultLinkAccountManager @Inject constructor(
                     consumerSession = consumerSessionSignup.consumerSession,
                     publishableKey = consumerSessionSignup.publishableKey,
                 )
+            }.mapCatching { account ->
```

Comment:
Yes it does actually. I added it because the old link attempted verification on look up, but this affects inline sign up. You're right, the verification screen will do same the think, so I'll take it out.